index.html

```diff
@@ -2495,6 +2571,15 @@
             <code>"NotSupportedError"</code> and abort these steps.
           </li>
           <li>
+            If the <var>options</var>.url is not an empty string and it is
+            not a valid <a>URL</a>, throw <code>"SyntaxError"</code> exception
+            and abort these steps. In addition, UA may check if
```

Comment:
@zolkis if scheme is not provided, it would be invalid url, right? I can change MAY to MUST if you want. Do we want to restrict, e.g., web apps running locally from filesystem?